app/src/features/auth/components/auth/forms/sign-up-form.tsx

```
@@ -0,0 +1,679 @@
+
+                                if (file)
+                                    handleAvatarChange(file);
+
+                                e.target.value = "";
+                            }}
+                            ref={fileInputReference}
+                            type="file"
+                        />
+
+                        <form.AppField
+                            children={() => (
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)